repos/fdbt-site/cypress_tests/cypress/support/steps.ts

```diff
@@ -279,6 +327,15 @@ export const completePeriodMultiServicePages = (
     completeMultipleProducts(numberOfProducts, multiProductNamePrefix, isCarnet);
 };
 
+export const completeCappedMultiServicePages = (
+): void => {
+    clickElementById('radio-option-multipleServices');
+    continueButtonClick();
+    randomlyChooseAndSelectServices();
+    continueButtonClick();
+    completeCappedDistanceJourney()
```

Comment:
completeCappedDistanceJourney() feels like it shouldnt be in here, because the function is called completeCappedMultiServicePages(). completeCappedGeoZonePages() is correct because it just fills out the geoZone specific pages.

Reply:
we can make it randomised or conditional once the rest of the journeys are added

Reply:
I'm not sure what you mean here @alainajaffer . I'm saying if the function says its filling out the multiService pages, it shouldnt also be filling out the pages *after* that.